packages/@react-spectrum/calendar/docs/Calendar.mdx

```diff
@@ -144,6 +144,19 @@ import {today} from '@internationalized/date';
 <Calendar aria-label="Event date" isDisabled />
 ```
 
+The `isDateDisabled` callback prop makes it possible to disable custom dates from the `Calendar`. It is being called for each date and if it returns `true`, the date cell is disabled.
+
```

Comment:
Mind updating the wording in the rest of the docs files (Calendar, RangeCalendar, DatePicker) to match DateRangePicker's change?

```suggestion
The `isDateDisabled` callback prop acts like a filter function. It is called for each visible date and if it returns `true`, the date cell is disabled.
```

Reply:
Updated it in all files 👌